lib/rubocop/cop/ezcater/rspec_dot_not_self_dot.rb

```diff
@@ -35,8 +35,8 @@ class RspecDotNotSelfDot < Cop
           %w(it specify example scenario).map { |identifier| "#{prefix}#{identifier}" }
         end.flatten + %w(its focus skip)).freeze
 
-        SELF_DOT_REGEXP = /\Aself\./.freeze
-        COLON_COLON_REGEXP = /\A(\:\:)/.freeze
+        SELF_DOT_REGEXP = /\Aself\./
+        COLON_COLON_REGEXP = /\A(\:\:)/
```

Comment:
Should we continue to explicitly freeze these for ruby 2.7 support?

Reply:
Sure, I can do that.